client/webpack.config.js

```diff
@@ -155,6 +155,7 @@ const webpackConfig = {
 		'entry-domains-landing': [ path.join( __dirname, 'landing', 'domains' ) ],
 		'entry-login': [ path.join( __dirname, 'landing', 'login' ) ],
 		'entry-gutenboarding': [ path.join( __dirname, 'landing', 'gutenboarding' ) ],
+		'entry-browsehappy': [ path.join( __dirname, 'landing', 'browsehappy' ) ],
```

Comment:
Out of curiosity, why is this `landing`? What's the difference between that and `document`?

Reply:
The component in `client/document` is ever ran only in the server, rendering the HTML document for the page. The component in `client/landing`, however, is part of the webpack client-side bundle for the entrypoint and runs mainly on the client. It runs on the server only when the page uses SSR.

`landing` is a directory for landing pages that have a separate entrypoint and are not part of the `entry-main` bundle.

Reply:
Thanks for explaining!